server/lib/merge-accounts.ts

```diff
@@ -80,7 +78,7 @@ export const simulateMergeAccounts = async (
 
   const addCountsToSummary = counts => {
     Object.entries(counts).forEach(([key, count]) => {
-      if (count > 0) {
+      if ((count as number) > 0) {
         addLineToSummary(`  - ${key}: ${count}`);
       }
     });
```

Comment:
Wouldn't it be better to type it in the function parameter? `as` kind of bypasses type checks

```es6
const addCountsToSummary = (counts: Record<string, number>) => {
```

Reply:
Note that this one doesn't seem to original from this PR, I'm curious why we haven't seen it before

Reply:
Changed the CollectiveFieldsConfig type as merge-accounts, maybe it propagated to here.